tests/plugins/dialog/manual/focusdialogtabhc.html

```diff
@@ -0,0 +1,11 @@
+<textarea id="editor"></textarea>
+
+<script>
+	( function() {
+		if ( bender.tools.env.mobile || !CKEDITOR.env.hc ) {
```

Comment:
Seems like just `.env.hc` is enough for ignoring test.
```suggestion
		if ( !CKEDITOR.env.hc ) {
```